lib/extend/filter.js

```diff
@@ -70,12 +67,11 @@ Filter.prototype.execSync = function(type, data, options = {}) {
   const filters = this.list(type);
   const ctx = options.context;
   const args = options.args || [];
-  let result;
 
   args.unshift(data);
 
   for (let i = 0, len = filters.length; i < len; i++) {
-    result = filters[i].apply(ctx, args);
+    const result = filters[i].apply(ctx, args);
```

Comment:
I'm not understanding why we need to change this? Due to performance?

Reply:
If that comment is about this line, the purpose is to reduce the scope of `result`.